connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -166,6 +169,13 @@ public void destroyConnector(final @PathParam("connector") String connector) thr
         completeOrForwardRequest(cb, "/connectors/" + connector, "DELETE", null);
     }
 
+    @PUT
+    @Path("/{connectorType}/config/validate")
```

Comment:
Do you suggest that we should use another resource instead of the connector resource? We can use another resource such as /connector_plugins/connectorType/config/validate.....